java/connector-node/risingwave-sink-cassandra/src/main/java/com/risingwave/connector/CassandraSink.java

```diff
@@ -193,7 +193,10 @@ public void drop() {
 
     private String createInsertStatement(String tableName, TableSchema tableSchema) {
         String[] columnNames = tableSchema.getColumnNames();
-        String columnNamesString = String.join(", ", columnNames);
+        String columnNamesString =
+                Arrays.stream(columnNames)
+                        .map(columnName -> "\"" + columnName + "\"")
+                        .collect(Collectors.joining(", "));
         String placeholdersString = String.join(", ", Collections.nCopies(columnNames.length, "?"));
         return String.format(
                 "INSERT INTO %s (%s) VALUES (%s)",
```

Comment:
All CQL identifiers, including table names, shall be double quoted to preserve case.
https://cassandra.apache.org/doc/latest/cassandra/developing/cql/definitions.html#identifiers

Maybe introduce a utility function / class for it?